deploy/testenv-deploy.js

```diff
@@ -0,0 +1,23 @@
+const env = require('./src/loadEnv')
+
+const deployToken = require('./src/utils/deployERC20Token')
+
+const mode = process.argv[2]
+
+async function main() {
+  //If other components of testing env needs to be initialized
+  //the script must be modified to handle the corresponding
+  //component through 'switch-case
```

Comment:
'switch-case'

Reply:
The same as https://github.com/poanetwork/poa-bridge-contracts/pull/103#discussion_r230857124